highway.tex

```diff
@@ -563,6 +563,21 @@ \subsection{Leader Selection and Weight Readjustment}
 
+\subsection{Independent Eras}
+
+Several aspects of the protocol are simplified by making different eras of the protocol essentially independent of each other. What this means in practice is that each era will be essentially an independent instantiation of the highway protocol with a fixed set of validators and weights and leader schedule (which are determined by the key block and booking block corresponding to that era). The genesis block of this era will have a state inherited from the end block of the previous era (the first block published in that era after that era's end time that is finalized to a sufficient degree of certainty).
+
+Although this view of the era structure simplifies several things, there are some disadvantages. Firstly, it is not clear how to reward or penalize behavior from the validators of the previous era for behavior that takes place after the creation of the final block. In particular, such rewards or penalties cannot be implemented in the previous era as the final block is the last block of that era whose values affect the next era, and therefore we will need some other mechanism. Another potential disadvantage is the time lost during the transition between eras. If the validators of the new era need to wait for the previous era's end block to be finalized, they may lose time before they are able to start.
+
+\subsubsection{Rewards and Penalties}
+
+In order to reward or penalize validators from the previous era for their tail behavior, we will need to implement new transactions that can be called in the new era that make reference to behavior in the previous era. For example, an accusation transaction provides evidence of an equivocation of a validator in the previous era and when executed has that validator's stake slashed accordingly.
+
+We will also need to have an initialization transaction, which includes evidence of a summit finalizing the end block of the previous era to a sufficient degree of certainty. When then transaction is processed, it causes the state of that end block to be adopted along with potentially rewards for the previous era's validators depending on their contributions to the finalization of the end block.
+
+\subsubsection{Parallel Eras}
+
+In order to avoid losing time between eras, we can allow a new era's validators to start creating blocks that order new transactions even before the last era's end block has been finalized. They will not be able to execute these transactions as the state of the system at the start of the era has not yet been determined. This means for example that a transaction to transfer money from $A$ to $B$ cannot be processed because it is not clear whether or not $A$ has sufficient money in their account. However, these transactions can be ordered so that once an initialization transaction is performed, the finalized transactions can be processed in order on the newly adopted state.
```

Comment:
To my mind, that initialization wouldn't be a _transaction_, but just any kind of evidence for finalization: the summit or the finality signatures. Once you have that locally, you can execute the new era's finalized blocks.